### PR TITLE
V18

### PR DESCRIPTION
**Pull request by jeffxor**
v18 is the next LTS version for Node. Created this PR for a new v18 build of this container.

### CODE REVIEW

File: .github/workflows/v18.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: sleavely/node-awscli:18.x
+
+on:
+  push:
+    branches:
+      - 'v16'
```
Comment (Sleavely) [inline .github/workflows/v18.yml]: This should probably be `v18`

File: .github/workflows/v18.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          pull: true
+          push: true
+          tags: ${{ github.workflow }},sleavely/node-awscli:latest
```
Comment (Sleavely) [inline .github/workflows/v18.yml:28]: Since version 18 will be `latest`, could you remove that tag from `v16.yml`?

**Review by Sleavely — CHANGES_REQUESTED** [review]
This is great, thank you! 🙌 

Before merging it would be great if you could address the review-comments and update the README with a reference to `v18` in the "Automatic updates" section.

**Comment by jeffxor** [comment]
@Sleavely should be good to merge. Thanks

**Comment by jeffxor** [comment]
@Sleavely anything else I need to do to get this merged

**Review by Sleavely — APPROVED** [review]

**Comment by Sleavely** [comment]
Looks good, thank you so much for taking the time.

**Pull request merged**